packages/editor/src/client/blocks/emoji-mart-override.css

```diff
@@ -0,0 +1,12 @@
+ .emoji-mart-bar:last-child {
```

Comment:
I think the best thing is to name the CSS file the same as the component it belongs to. I suggest you move the emoij-mart-picker to its own component and call the css file the same. 

Also make the css file a less file and import the less file from the component inside that less file.

Reply:
That sounds good, will implement.